### DISCUSSION TITLE
Maybe I'm missing it...

### DISCUSSION THREAD

**Discussion by bonelifer** [jcorporation/myMPD#1332] [Q&A] [answered]
I would like a view where, when you click on an album, the tracks are displayed at the bottom, allowing you to go to an album and queue tracks from there. Is this already an option that I can enable by changing settings?"

**Comment by jcorporation** [comment, marked as answer]
You mean the database view? It displays all albums with covers and opens a detailed view with the tracks for this album. 

![Screenshot from 2024-07-19 01-16-09](https://github.com/user-attachments/assets/1096a6b8-f34b-4d60-adb7-d7eb293960a4)
